docs/tutorial/part-three/index.md

```diff
@@ -208,17 +208,17 @@ export default () => (
 
 ![with-layout2](with-layout2.png)
 
-Sweet, the layout is working! The content of your index page is still centered.
+लेआउट काम कर रहा है! आपके अनुक्रमणिका पृष्ठ की सामग्री अभी भी केंद्रित है।
 
-But try navigating to `/about/`, or `/contact/`. The content on those pages still won't be centered.
+लेकिन `/about/`, या `/contact/` के लिए नेविगेट करने का प्रयास करें। उन पृष्ठों की सामग्री अभी भी केंद्रित नहीं होगी।
 
-4. Import the layout component in `about.js` and `contact.js` (as you did for `index.js` in the previous step).
+4. लेआउट कौम्पोनॅन्टस को `about.js` और` contact.js` में आयात करें (जैसा कि आपने पिछले चरण में `index.js` के लिए किया था)।
 
-The content of all three of your pages is centered thanks to this single shared layout component!
+आपके सभी तीन पृष्ठों की सामग्री इस एकल साझा लेआउट कौम्पोनॅन्टस के लिए धन्यवाद केंद्रित है!
```

Comment:
`आपके सभी तीन पृष्ठों की सामग्री इस एकल साझा लेआउट कौम्पोनॅन्टस के लिए धन्यवाद केंद्रित है!` => `एक अकेले शेयर्ड लेआउट कौम्पोनॅन्ट से सरे तीनो पेजेज के कंटेंट सेण्टर में हो गए हैं!`

Reply:
`से सरे तीनो` => `से तीनो`

sorry about the miss here